scripts/xtensa-build-zephyr.sh

```diff
@@ -133,6 +138,7 @@ west_init_update()
 
 	# zephyr_fetch_and_switch    origin   pull/38374/head
 	# zephyr_fetch_and_switch    origin   19d5448ec117fde8076bec4d0e61da53147f3315
+	zephyr_fetch_and_switch    sof       sof/stable-v2.1
```

Comment:
This makes it impossible to clone any other Zephyr version. This hack here was never meant to be merged, only submitted. Let me submit something more flexible today.

Reply:
> Let me submit something more flexible today.

Done in
- #5530

So the idea is to cherry-pick #5530 to stable-v2.1 once it's merged and then change only 2 lines / 2 default values on top.